### PR TITLE
#767: Another code cleanups

### PR DESCRIPTION
**Pull request by bonkey**
This time not much touched

### CODE REVIEW

**Comment by bonkey** [comment]
👍

File: AtlasSDK/AtlasSDK/API/AtlasAPI_configure.swift [hunk 1]
```diff
@@ -18,7 +18,8 @@ extension AtlasAPI {
 
      - Parameters:
          - options: Options for API client to be created. When `nil`, `$INFOPLIST_FILE` file of the app is used as configuration.
-         - completion: Fired when network configuration call is finished. Containts `Result.success` with `AtlasAPI` or `Result.failure` with `Error` reason.
+         - completion: Fired when network configuration call is finished.
+            Containts `Result.success` with `AtlasAPI` or `Result.failure` with `Error` reason.
```
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/API/AtlasAPI_configure.swift]: `Contains`

**Comment by hani-ibrahim** [comment]
👍

File: AtlasSDK/AtlasSDK/API/AtlasAPI_configure.swift (hunk 1 above)
Comment (bonkey) [inline AtlasSDK/AtlasSDK/API/AtlasAPI_configure.swift]: OK, will be in `issue-33` or another `issue-767` 👺

**Comment by bonkey** [comment]
New stuff here 28c5c98 😋

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
How do you like that (not all finished) f37fa48

**Comment by bonkey** [comment]
WTF? 

screen shot 2017-03-02 at 17 21 25

**Comment by bonkey** [comment]
and another de733c9

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**